docs/content/en/docs/packages/overview.md

```diff
@@ -42,3 +49,38 @@ Please check out [package management]({{< relref "../packages/packages" >}}) for
 
 ### Curated packages artifacts
 There are three types of build artifacts for packages: the container images, the helm charts and the package bundle manifests. The container images, helm charts and bundle manifests for all of the packages will be built and stored in EKS Anywhere ECR repository. Each package may have multiple versions specified in the packages bundle. The bundle will reference the helm chart tag in the ECR repository. The helm chart will reference the container images for the package.
+
+
+### Installing packages on workload clusters
+
+![Installing packages on workload clusters](/images/packages-controller-workload-cluster.png)
+
+The packages controller only runs on the management cluster. The curated packages controller determines which cluster to install your package on based on the namespace you create the `Package` resource in.
+
```

Comment:
Can we move up the `Package` spec example here? I feel like this is where it becomes relevant.

After the example, we can also add some text referencing the example to conclude the point, "The package controller would install the `my-hello-eks-anywhere` package in the  workload cluster `wk0`